ale_linters/javascript/flow.vim

```diff
@@ -21,11 +21,15 @@ endfunction
 
 function! ale_linters#javascript#flow#GetCommand(buffer) abort
   return ale_linters#javascript#flow#GetExecutable(a:buffer)
-  \   . ' check-contents --respect-pragma --json --from ale'
+  \   . ' check-contents --respect-pragma --json --from ale %s'
```

Comment:
Will this not run the linter on the file, instead of the buffer data? That would make it so you'll be getting the errors for the file as it is on disk, not as you are editing it.